pkg/core/blockchain.go

```diff
@@ -562,6 +562,20 @@ func (bc *Blockchain) GetAssetState(assetID util.Uint256) *AssetState {
 	return as
 }
 
+// GetAccountState returns the account state from its script hash
+func (bc *Blockchain) GetAccountState(scriptHash util.Uint160) *AccountState {
+	var as *AccountState
+	bc.Store.Seek(storage.STAccount.Bytes(), func(k, v []byte) {
+		var a AccountState
+		a.DecodeBinary(bytes.NewReader(v))
```

Comment:
```go
		var a AccountState
		if err := a.DecodeBinary(bytes.NewReader(v)); err == nil && a.ScriptHash == scriptHash {
			as = &a
		}
```

what do you think if we fix it like this?

Reply:
:+1: